### ISSUE TITLE
Adding New Tables in a dotmim syncsetup

### ISSUE THREAD

**Issue by ramyscard** [Mimetis/Dotmim.Sync#512]
hello,

i am testing the DOTMIM framework and it looks great .

i have an issue when adding new table to the setup for example "AddressesTB "
i am using WebClient to sync my databases. 
the server side is Mysql and the client side is SqlServer using ChangeTracking provider.

im getting an error saying Could not find stored procedure 'dbo.DOTMIMAddressesTB_changes'.
shouldn't it be created automatically when adding new tabels?

Regards,
Ramy

**Comment by Mimetis** [comment]
Indeed, you found a bug in the automatic migration.
It's fixed and will be published in the next pre-release of `v0.7.3`
Thanks a ton @ramyscard  !

That being said, It only affects the client side and not the server side.

As a workaround, what you can do as @maxrodro is mentioning, is to use `DeprovisionAsync()` on the client.
Here is an example you can use until the new version is released:

``` cs
// Get the client scope containing the setup used on the client
var clientScope = await agent.LocalOrchestrator.GetClientScopeAsync();

// If no setup is present, we have a new database here, so no need to check server
if (!clientScope.IsNewScope)
{
    // Get the server scope containing the server setup
    // (and potentially more or less tables)
    var serverScope = await agent.RemoteOrchestrator.GetServerScopeAsync();

    // Use special method HasSameStructure to compare both setup
    // if setups are different, just Deprovision client and make
    // a new full sync on next SynchronizeAsync()
    if (!serverScope.Setup.HasSameStructure(clientScope.Setup))
        await agent.LocalOrchestrator.DeprovisionAsync();
}

var s = await agent.SynchronizeAsync(progress);
Console.WriteLine(s);
```

**Comment by ramyscard** [comment]
Thanks alot @Mimetis  , @maxrodro  . and a big thanks for this awesome framework.
